tests/integration/test_fim/test_synchronization/data/test_cases/cases_sync_overlap.yaml

```diff
@@ -0,0 +1,111 @@
+- name: Doubled sync_interval smaller than max_interval
+  description: Detect that sync_interval is doubled when a new sync tries to start before sync has ended
+  configuration_parameters:
+    INTERVAL: 3
+    RESPONSE_TIMEOUT: 3
+    MAX_INTERVAL: 20
+  metadata:
+    interval: 3
+    response_timeout: 3
+    doubled_times: 1
+    max_interval: 20
+    files: 5
+    lower: false
+
+- name: Doubled sync_interval capped at max_interval
+  description: Detect that when doubled sync_interval will not exceed max_interval
+  configuration_parameters:
+    INTERVAL: 5
+    RESPONSE_TIMEOUT: 5
+    MAX_INTERVAL: 8
+  metadata:
+    interval: 5
+    response_timeout: 5
+    doubled_times: 1
+    max_interval: 8
+    files: 10
+    lower: false
+
+- name: response_timeout lower than interval
+  description: Detect behavior when response_timeout is lower than interval
+  configuration_parameters:
+    INTERVAL: 5
+    RESPONSE_TIMEOUT: 3
+    MAX_INTERVAL: 8
+  metadata:
+    interval: 5
+    response_timeout: 3
+    doubled_times: 1
+    max_interval: 8
+    files: 10
+    lower: false
+
+- name: Invalid response_timeout value
+  description: Check behavior when invalid (string) value is configured for response_timeout
+  configuration_parameters:
+    INTERVAL: 3
+    RESPONSE_TIMEOUT: invalid
+    MAX_INTERVAL: 20
+  metadata:
+    interval: 3
+    response_timeout: invalid
+    doubled_times: 3
+    max_interval: 20
+    files: 5
+    lower: false
```

Comment:
I'd like to change the value of `RESPONSE_TIMEOUT `to '1' so that it thinks q is a string, but not the `invalid` word.

Reply:
Tested using a number with quotes as a string. The quotes are parsed and removed, and it is configured correctly. in order to use a string passing a string like this is the clearer option.